src/styles/post-template.less

```diff
@@ -10,6 +10,19 @@
 @designColor: #ab59c6;
 @wtfColor: #ff00af;
 
+@baseFontSize: 14px;
+// Modular scale: Major second
+@ratioSm: 1.125em;
+// Title font size is two steps up in the scale
+@titleFontSizeSm: pow(@ratioSm, 2);
```

Comment:
I don't understand this -- it seems like you're calculating a new font size, but the code is setting titleFontSizeSm to `1.125^2`?  Shouldn't this be `@baseFontSize * pow(@ratioSm, 2)` or something?  From your test plan it sounds like the current code works, but I don't understand why.

Reply:
This works because I'm using `em` units for the title font size, which are relative to the base font size.

Reply:
Ah, TIL.